images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -160,3 +160,31 @@ Hashtable for service arguments
         }
     }
 }
+
+function Get-VS19ExtensionVersion
+{
+    param (
+        [string] [Parameter(Mandatory=$true)] $packageName
+    )
+
+    $vsProgramData = Get-Item -Path "C:\ProgramData\Microsoft\VisualStudio\Packages\_Instances"
+    $instanceFolders = Get-ChildItem -Path $vsProgramData.FullName
+
+    if($instanceFolders -is [array])
```

Comment:
nitpick: missing space `if ($instanceFolders -is [array])`

Reply:
thanks! Fixed